docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -17,59 +17,31 @@ include::./status-include.adoc[]
 
 You can install the Quarkus CLI via:
 
-* <<homebrew,Homebrew>>
-* <<jbang,JBang>>
-* <<sdkman,SDKMAN!>>
+* <<jbang,JBang>> for Linux, OSX and Windows (no java required)
+* <<sdkman,SDKMAN!>> for Linux and OSX (java required)
+* <<homebrew,Homebrew>> for OSX (java required)
 
```

Comment:
While I agree with moving Homebrew down, I would perhaps leave SDKMan first unless you have resolved the trusted source issue with JBang, as that is confusing users.

Have you fixed the warning for JBang (re: maven central as source or some other means)?

Reply:
I decided to make them alphabetically sorted. And I would prefer we keep it that way. There's absolutely no way we can find an ordering that will fit all the users so alphabetical is probably the best.

Reply:
Is it OSX or macOS? Also make sure Java is capitalized

Reply:
I agree with @ebullient on leaving SDKMan first. IMHO we should promote what works better and faster first instead of alphabetically.